### ISSUE TITLE
Project Translation

### ISSUE THREAD

**Issue by shuvashish76** [slowscript/warpinator-android#104]
> https://github.com/slowscript/warpinator-android#translations

I've seen OSS projects use these sites for translation purpose
- [Crowdin](https://crowdin.com/)
- [Transifex](https://www.transifex.com/)
- [Weblate](https://weblate.org/en/)

I think Weblate is better as it's [opensource](https://github.com/WeblateOrg/weblate).

Features: https://weblate.org/en/features/
![Screenshot_2021-12-05-21-15-09](https://user-images.githubusercontent.com/48226312/144753433-4fc61e84-f12d-42d0-885e-1b9abd399061.jpeg)
https://weblate.org/en/hosting/

**Comment by cardpuncher** [comment]
I second shuvashish76's proposal. Such a website makes things easier for translators, you don't have to deal with git, xml files or diffs to find what got added. And my preference would go to Weblate as well, since it's free software and is nice to use.

**Comment by slowscript** [comment]
I'd favor using a free software solution as well, but my issue with Weblate is that in other projects that I've seen use it, it creates way too many commits and clutters the history. Not sure if it's a configuration issue, but I'd prefer to do a single "Import translations" commit before every release.

**Comment by shuvashish76** [comment]
IDK how Weblate works.

> creates way too many commits and clutters the history...I'd prefer to do a single "Import translations" commit before every release

Ping @comradekingu for help

**Comment by comradekingu** [comment]
The [cadence of PRs](https://docs.weblate.org/en/latest/admin/projects.html#component-commit-pending-age) can be set in the project settings, or it can all be implemented with a manual push once ready.

**Comment by liberii** [comment]
I have translated "warpinator-android" into Italian, but I don't know how to send the translation since the values-it folder is missing. If the project was on Crowdin or Transifex it would have been easy for me.

**Comment by comradekingu** [comment]
The frequency can be set lower, or omitted totally.
Add "kingu" there and I can help.

29. mai 2022 13:20:44 Shuvashish Sahoo ***@***.***>:

**Comment by slowscript** [comment]
@comradekingu Thanks for the offer of help. I have played with Weblate a bit locally on my PC and last week I've created the project on hosted Weblate. I've already done most of the setup I want (squash everything to one commit, push manually before release). Do I need to resolve all the items on the community localisation checklist in order to get approved? I'm also not sure whether safe-html should be enabled when I want HTML to be kept in the translation. It creates a warning I don't know how to get rid of.
